Tests/test_file_gbr.py

```diff
@@ -0,0 +1,15 @@
+from helper import unittest, PillowTestCase
+
+from PIL import Image, GbrImagePlugin
+
+
+class TestFileGbr(PillowTestCase):
+
+    def test_invalid_file(self):
+        self.assertRaises(SyntaxError, lambda: GbrImagePlugin.GbrImageFile("Tests/images/flower.jpg"))
```

Comment:
Depending on where the indent is, that can cause flake8 under-or-over-intended warnings. I've added another commit to fix flake8 warnings from the first commit, and a few other Tests as well.